### PR TITLE
Remove Bootstrap and make required changes

### PR DESCRIPTION
**Pull request by samwilson**
This removes all of Bootstrap and updates everything to look much
the same as it did with Bootstrap. Not quite exactly the same.

### CODE REVIEW

File: assets/app.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    max-width: var( --width-breakpoint-desktop-wide );
+    padding: 1rem;
+}
+
+main {
+    background-color: var( --background-color-base );
+}
+
+aside.tutorial {
+    border-radius: 2px;
```
Comment (prtksxna) [inline assets/app.scss]: `--border-radius-base`

File: assets/app.scss [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    border: 1px solid var(--wmui-color-base50);
```
Comment (prtksxna) [inline assets/app.scss]: `--border-base`

**Comment by samwilson** [comment]
Weird `:root` border issue seems to be this one: https://github.com/webpack-contrib/css-loader/issues/708

**Comment by samwilson** [comment]
Fixed bugs and added hacky workaround for the border bug. Updated staging site https://tools.wmflabs.org/svgtranslate-test/

**Comment by samwilson** [comment]
Sorry... ignore me. I forgot to fix the border problem elsewhere.

File: assets/app.scss (hunk 2 above)
Comment (samwilson) [reply assets/app.scss]: By 'done' I of course meant 'not done'... :)

Done *now* though.

**Comment by samwilson** [comment]
Okay, this is ready for review.

File: assets/app.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        margin: auto;
+    }
+    li {
+        font-size: 1rem;
+        flex: 1;
+        display: flex;
+        align-items: center;
+    }
+    p.image {
+        flex: 1;
```
Comment (mooeypoo) [inline assets/app.scss]: Flex is actually only partially supported in Edge (see https://caniuse.com/#search=flex )
We avoid using it in MW, but the target audience of this product is different...  We should make sure we test this in IE10/11 and see if we need to redo the approach.

File: templates/base.html.twig [hunk 4]
```diff
@@ -1,83 +1,69 @@
 <!DOCTYPE html>
-<html>
+<html lang="{{ lang() }}">
```
Comment (mooeypoo) [inline templates/base.html.twig]: Can we get dir= easily? If yes, add it in. If no, don't bother, but let's add that as a todo when we add proper language support with ULS.

Comment (samwilson) [reply templates/base.html.twig]: I've added it.

File: assets/app.scss (hunk 3 above)
Comment (samwilson) [reply assets/app.scss]: I thought flex was okay, it was just css grid that we're avoiding. But you're right, there's likely to be problems. (Side note: we're using flex in templatewizard!)

I don't have access to edge. Should I come up with a float-only way of doing this same layout? Or should we just stick with bootstrap's grid (we can include just it)?

**Comment by samwilson** [comment]
This is ready for re-review. Switched to LESS and fixed the `dir`. Didn't change flex yet.

File: i18n/en.json [hunk 5]
```diff
@@ -20,6 +20,7 @@
   "update-commons-more": "or download directly to your computer.",
   "login": "Log in",
   "logout": "Log out",
+  "language-settings": "Language settings",
```
Comment (MusikAnimal) [inline i18n/en.json]: qqq?

**Comment by mooeypoo** [comment]
Looks good to me **Except** for the missing qqq that @MusikAnimal points out. Let's not wait with those to a followup, they're fairly easy to creat -- and to forget if we don't do it now :)

When those are done I'm good with merging this.

**Comment by samwilson** [comment]
I've added the missing qqq message and will make a separate PR for all the others.

**Pull request merged**